spec/integration/admin/grove_monitor_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+include Pages::Authentication
+
+RSpec.feature 'Grove Monitor' do
+  let(:teacher) { create(:teacher) }
+  let(:grove_monitor_page) { Pages::GroveMonitorPage.new }
+  let(:dashboard_page) { Pages::DashboardPage.new }
+
+  describe 'as a teacher' do
+    before { login(teacher) }
+
+    it 'can be reached from the main dashboard' do
+      dashboard_page.click_on("Grove Monitor")
+      expect(current_path).to eq(admin_grove_monitor_path)
+    end
+
+    it 'provides navigation links' do
+      expect(grove_monitor_page).to have_content("Teacher Dashboard")
```

Comment:
If you want to test this, and feel its important, lets pull the construction of the data that backs the navigation into a helper or helper class that takes in a user and yields the data to build the links. something like 
`@navigation = NavigationBuilder.new(user) #=> {'Teacher Dashboard' => '/admin/dashboard/, ...}` and then use an enumarble to iterate over and construct the navigation within the erb template.